recipe2023/index.html

```diff
@@ -1,80 +1,103 @@
 <!DOCTYPE html>
 <head>
-  <title>Yolis' Recipe Page</title>
+  <title class="main-title">Yolis' Recipe Page</title>
```

Comment:
```suggestion
  <title>Yolis' Recipe Page</title>
```

anything inside of `<head>` is not visible on the page and therefore doesn't have support for classes.